cli/gardener_ci/whitesource_cli.py

```diff
@@ -8,15 +7,17 @@ def upload_and_scan_component(
     whitesource_cfg_name: str,
     component_descriptor_path: str,
     requester_mail: str,
-    component_name: str,
-    extra_whitesource_config: str = {},
-    notification_recipients: [str] = [],
     cve_threshold: float = 5.0,
+    notification_recipients: str = '[]',
```

Comment:
in fact, cli.py does support lists, like so: `[str]` (admittedly, it would be more aligned w/ the `typing` module, if the implementation would instead rather accept `typing.List[str]`)